docs/en/guides/webhooks.md

```diff
@@ -189,6 +189,16 @@ A consumer service must respond to deliveries in order for the system to conside
 
 {% include warning title="Response timeouts" content="A response must be received within six seconds of delivery to a webhook’s URL, after which the connection will be closed. Failure to respond in time will result in a failed delivery." %}
 
+#### Throttling
+
+Your consumer response times to deliveries will impact webhooks throughput for your site.
+Each site is allowed 1 minute of response time per minute. So if all configured consumer endpoints for a site take the full 6 seconds to respond, webhooks deliveries for that site will be throttled to 10 per a minute.
+
+Where a high rate of overall throughput is needed, then consumer endpoints should be designed according to the following model:
+ 1. Receive the request
+ 2. Spawn another process/thread to handle it the way you want
+ 3. Answer an acknowledging 200 immediately
+
```

Comment:
Possibly worth noting here that the UI allows you to see the response times of the webhook endpoint?